tensorflow/lite/delegates/nnapi/nnapi_delegate.cc

```diff
@@ -57,6 +57,7 @@ limitations under the License.
 #include "tensorflow/lite/nnapi/nnapi_implementation.h"
 #include "tensorflow/lite/nnapi/nnapi_util.h"
 #include "tensorflow/lite/util.h"
+#include "tensorflow/lite/schema/schema_generated.h"
```

Comment:
let's wrap it with "#ifdef NNAPI_VERBOSE_VALIDATION"

Reply:
Thanks @terryheo, Done